codegenerator/cli/npm/envio/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "envio",
+  "version": "local",
+  "private": true,
```

Comment:
Added private so it's not published by mistake. Otherwise it will be overwritten in ci